spec/integration/lhm_spec.rb

```diff
@@ -283,6 +283,38 @@
       end
     end
 
+    it 'should fail if the triggers do not exist after copying all rows' do
+      table_create(:users)
+
+      execute("INSERT INTO users (username) VALUES ('a user')")
+
+      Lhm::Invoker.any_instance.stubs(:triggers_still_exist?).returns(false)
+
+      exception = assert_raises do
+        Lhm.change_table(:users) do |t|
+          t.rename_column(:group, :fnord)
+        end
+      end
+
+      assert_match "Required triggers do not exist", exception.message
+    end
+
+    it 'should not perform the table rename if the triggers do not exist after copying all rows' do
+      table_create(:users)
+
+      execute("INSERT INTO users (username) VALUES ('a user')")
+
+      Lhm::Invoker.any_instance.stubs(:triggers_still_exist?).returns(false)
+
+      Lhm::LockedSwitcher.any_instance.expects(:run).never
```

Comment:
Same here... Is an assert to that method is the most effective? Testing the interface not implementation etc etc?